tests/sentry/lang/javascript/test_processor.py

```diff
@@ -118,7 +118,6 @@ def test_tag_suspected_console_error(self):
         release = self.create_release(project=project, version="12.31.12")
 
         data = {
-            "is_exception": True,
```

Comment:
Is this not necessary anymore?

Reply:
This (and the other, similar spot) are what I was talking about in the last bit of the PR description:

> I came across two tests which included `is_exception` values in a spot they never show up IRL, being passed to functions which never check them.